<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <item>
            <property name="text">
             <string>REFRESH</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/new/prefix1/katynko/refresh.png</normaloff>:/new/prefix1/katynko/refresh.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>&amp;Open</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_2">
           <property name="text">
            <string>C</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_3">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/new/prefix1/katynko/знімок7.png</normaloff>:/new/prefix1/katynko/знімок7.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_4">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/new/prefix1/katynko/info.png</normaloff>:/new/prefix1/katynko/info.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Advanced Serial Port Settings</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Data bits</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_3"/>
            </item>
            <item row="0" column="2" rowspan="2">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_3">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Parity</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QComboBox" name="comboBox_5"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Stop bits</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox_4"/>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Flow control</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QComboBox" name="comboBox_6"/>
            </item>
            <item row="2" column="0" colspan="2">
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>132</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="4">
             <widget class="QPushButton" name="pushButton_2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>&amp;Apply</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Send Settings</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Enter</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_7">
              <item>
               <property name="text">
                <string>CR+LF</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>LF</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="sizeConstraint">
               <enum>QLayout::SetFixedSize</enum>
              </property>
              <item>
               <widget class="QLineEdit" name="lineEdit_2">
                <property name="text">
                 <string>\</string>
                </property>
                <property name="maxLength">
                 <number>1</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_8">
                <property name="currentIndex">
                 <number>3</number>
                </property>
                <item>
                 <property name="text">
                  <string>CR+LF</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>CR</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>LF</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>+/&amp;&amp;</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Server</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0" colspan="2">
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="text">
               <string>127.0.0.1</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QSpinBox" name="spinBox">
              <property name="prefix">
               <string>port </string>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>1234</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="checkBox_2">
              <property name="text">
               <string>Auto connect</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QToolButton" name="toolButton_5">
              <property name="text">
               <string>c/d</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>AT command</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="leHost">
           <property name="text">
            <string>127.0.0.1</string>
           </property>
           <property name="maxLength">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbPort">
           <property name="prefix">
            <string>port </string>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>1234</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbConnect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDisconnect">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_9">
           <item>
            <property name="text">
             <string>CR+LF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_6">
           <property name="text">
            <string>C</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_2">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSend">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
